instrumentation/logback/logback-access-jetty-12.0/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/logback/access/jetty/v12_0/TestAccessLogWithJetty.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.logback.access.jetty.v12_0;
+
+import static io.opentelemetry.javaagent.instrumentation.logback.access.jetty.v12_0.AccessEventMapper.ACCESS_EVENT_NAME;
+import static io.opentelemetry.sdk.testing.assertj.OpenTelemetryAssertions.equalTo;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import ch.qos.logback.access.common.spi.Util;
+import ch.qos.logback.access.jetty.RequestLogImpl;
+import ch.qos.logback.core.testUtil.RandomUtil;
+import io.opentelemetry.api.logs.Severity;
+import io.opentelemetry.instrumentation.testing.junit.AgentInstrumentationExtension;
+import io.opentelemetry.semconv.ClientAttributes;
+import io.opentelemetry.semconv.HttpAttributes;
+import io.opentelemetry.semconv.ServerAttributes;
+import io.opentelemetry.semconv.UrlAttributes;
+import io.opentelemetry.semconv.UserAgentAttributes;
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+class TestAccessLogWithJetty {
+
+  static RequestLogImpl requestLogImpl = new RequestLogImpl();
+  static JettyServer jettyFixture;
+  static String jettyFixtureUrlStr;
+
+  static final int RANDOM_SERVER_PORT = RandomUtil.getRandomServerPort();
```

Comment:
Ideally you let the server bind to port `0` and later ask what was the actual port the os gave it. If that is not an option then use our `PortUtils.findOpenPort()`. `RandomUtil.getRandomServerPort()` just picks a random number, it may collide with a port that is used in some other test.